tests/test_templates.py

```diff
@@ -624,7 +636,19 @@ class TestGradientIntegration:
                                          (QAOAEmbedding,
                                           {'features': [1., 2.], 'weights': [[0.1, 0.1, 0.1]]},
                                           {'wires': range(2)},
-                                          [1])
+                                          [1]),
```

Comment:
Same comment as for the previous test case

Reply:
See answer above...